pkg/cmd/clustertask/describe_test.go

```diff
@@ -360,3 +360,67 @@ func TestClusterTaskV1beta1_custom_output(t *testing.T) {
 		t.Errorf("Result should be '%s' != '%s'", got, expected)
 	}
 }
+
+func TestClusterTaskDescribe_With_Results(t *testing.T) {
+	clock := clockwork.NewFakeClock()
+
+	clustertasks := []*v1beta1.ClusterTask{
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:              "clustertask-1",
+				CreationTimestamp: metav1.Time{Time: clock.Now().Add(-1 * time.Minute)},
+			},
+			Spec: v1beta1.TaskSpec{
+				Description: "a clustertest description",
+				Steps: []v1beta1.Step{
+					{
+						Container: corev1.Container{
+							Name:  "hello",
+							Image: "busybox",
+						},
+					},
+				},
+				Results: []v1beta1.TaskResult{
+					{
+						Name:        "result-1",
+						Description: "This is a description for result 1",
+					},
+					{
+						Name:        "result-2",
+						Description: "This is a description for result 2",
+					},
+					{
+						Name: "result-3",
+					},
+				},
+			},
+		},
+	}
+
+	ns := []*corev1.Namespace{
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "ns",
+			},
+		},
+	}
+
+	version := "v1beta1"
+	tdc := testDynamic.Options{}
+	dynamic, err := tdc.Client(
+		cb.UnstructuredV1beta1CT(clustertasks[0], version),
+	)
+	if err != nil {
+		t.Errorf("unable to create dynamic client: %v", err)
+	}
+
+	cs, _ := test.SeedV1beta1TestData(t, pipelinev1beta1test.Data{ClusterTasks: clustertasks, Namespaces: ns})
```

Comment:
Namespaces can be removed:

```suggestion
	cs, _ := test.SeedV1beta1TestData(t, pipelinev1beta1test.Data{ClusterTasks: clustertasks})
```